dat/main.py

```diff
@@ -65,6 +65,11 @@ def write_generated_reference_tables(table_names, output_path):
     logging.info('Generated reference tables successfully written')
 
 
+@click.command()
+def write_generated_reference_table_3():
+    create_reference_tables.create_reference_table_3()
```

Comment:
does this not work with the existing framework? @edmondo1984 - is it documented/clear/easy to add more reference tables?

also - from an API perspective - do we want users to be able to generate just a single reference table? I think so.

@edmondo1984 is there an existing API for that?

Reply:
https://github.com/delta-incubator/dat/pull/17/files/385b3822570fcec23d7e04a577f938ed8f6ef682..83f3e7e1b4a65ad68c5ff67fb18a476c9e2dbeca#diff-43f89dd411632a233f2c80cadaf9c223cbe98f2bb6bf5cad6f74f01d811b16c3R39

if someone would specify --table-name=foo, the only table name with table foo would be specified